build.gradle

```diff
@@ -70,13 +70,15 @@ configurations {
 
 sourceSets {
     main  {
-        if (Integer.parseInt(ideaVersion.substring(3).substring(0, 4)) >= 2021) {
+        if (ideaVersion == "LATEST-EAP-SNAPSHOT"
```

Comment:
I'm newbie with Gradle, but it looks like tit is Java code, should be not:
 
`"LATEST-EAP-SNAPSHOT".equals(ideaVersion)` ?

Reply:
This is `groovy`, i.e. less verbose Java ;-). `==` is similar to calling "equals", so this is fine.
https://stackoverflow.com/a/9682512/753170

Reply:
it's groovy. In groovy `.equals()` is `==`. Reference equality in groovy is denoted with `===`
See https://docs.groovy-lang.org/latest/html/documentation/core-operators.html